tests/suite-state/06-format.t

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 # THIS FILE IS PART OF THE CYLC SUITE ENGINE.
-# Copyright (C) 2008-2019 NIWA & British Crown (Met Office) & Contributors.
+# Copyright (C) NIWA & British Crown (Met Office) & Contributors.
 #
```

Comment:
Copied from the PR to master... we should probably update copyright statements on 7.8.x, damn it.

Reply:
See #3605